ElvargServer/src/main/java/com/elvarg/game/entity/impl/player/Player.java

```diff
@@ -93,6 +94,8 @@
 import io.netty.buffer.ByteBuf;
 
 public class Player extends Mobile {
+
+	public static PersistenceMethod Persistence = PersistenceMethod.getSingleton();

```

Comment:
```suggestion
	public static PersistenceMethod persistence = PersistenceMethod.getSingleton();

```
Stick to java naming convention

Reply:
Removed as moved to global constant in GameConstants